README.md

```diff
@@ -18,29 +18,30 @@ your own Truffle language.
 
 * Clone SL repository using
   `git clone https://github.com/graalvm/simplelanguage`
-* Download Graal VM Development Kit from 
+* Download Graal VM Development Kit (verson 0.31) from 
```

Comment:
typo. You cannot download an older version from OTN. I'd rather not mention the version here. Also typo.